sdk/ai/Azure.AI.Projects/CHANGELOG.md

```diff
@@ -1,17 +1,13 @@
 # Release History
 
-## 1.0.0-beta.8 (Unreleased)
+## 1.0.0-beta.8 (2025-04-22)
 
```

Comment:
Weren't there other changes related to Agent as a tool, similar to what Glenn had for Python? (other than sample update)

Reply:
There are not, the ConnectedAgentToolDefinition constructor doesn't require customization.

Reply:
We do have new sample?